docs/docsite/rst/inventory_guide_iocage_dhcp.rst

```diff
@@ -0,0 +1,178 @@
+   | 205 | srv_2 | off  | up    | jail | 14.2-RELEASE-p3 | epair0b|10.1.0.204 | -   | ansible_client | no       |
+   +-----+-------+------+-------+------+-----------------+--------------------+-----+----------------+----------+
+   | 206 | srv_3 | off  | up    | jail | 14.2-RELEASE-p3 | epair0b|10.1.0.169 | -   | ansible_client | no       |
+   +-----+-------+------+-------+------+-----------------+--------------------+-----+----------------+----------+
+
+Create the inventory configuration. Use the option :ansopt:`community.general.iocage#inventory:sudo`
+
+.. code-block:: console
+
+   shell> cat hosts/02_iocage.yml
+
+.. code-block:: yaml
+
+   plugin: community.general.iocage
+   host: 10.1.0.73
+   user: admin
+   sudo: true
+
+Display the inventory:
+
+.. code-block:: console
+
+   shell> ansible-inventory -i hosts/02_iocage.yml --list --yaml
+   all:
+     children:
+       ungrouped:
+         hosts:
+           srv_1:
+             iocage_basejail: 'no'
+             iocage_boot: 'off'
+             iocage_ip4: 10.1.0.183
+             iocage_ip4_dict:
+               ip4:
+               - ifc: epair0b
+                 ip: 10.1.0.183
+                 mask: '-'
+               msg: ''
+             iocage_ip6: '-'
+             iocage_jid: '204'
+             iocage_release: 14.2-RELEASE-p3
+             iocage_state: up
+             iocage_template: ansible_client
+             iocage_type: jail
+           srv_2:
+             iocage_basejail: 'no'
+             iocage_boot: 'off'
+             iocage_ip4: 10.1.0.204
+             iocage_ip4_dict:
+               ip4:
+               - ifc: epair0b
+                 ip: 10.1.0.204
+                 mask: '-'
+               msg: ''
+             iocage_ip6: '-'
+             iocage_jid: '205'
+             iocage_release: 14.2-RELEASE-p3
+             iocage_state: up
+             iocage_template: ansible_client
+             iocage_type: jail
+           srv_3:
+             iocage_basejail: 'no'
+             iocage_boot: 'off'
+             iocage_ip4: 10.1.0.169
+             iocage_ip4_dict:
+               ip4:
+               - ifc: epair0b
+                 ip: 10.1.0.169
+                 mask: '-'
+               msg: ''
+             iocage_ip6: '-'
+             iocage_jid: '206'
+             iocage_release: 14.2-RELEASE-p3
+             iocage_state: up
+             iocage_template: ansible_client
+             iocage_type: jail
+
+Note: If the option :ansopt:`community.general.iocage#inventory:env` is used and :ansopt:`community.general.iocage#inventory:sudo` is enabled, enable also :ansopt:`community.general.iocage#inventory:sudo_preserve_env`. For example,
+
+.. code-block:: yaml
+
+   plugin: community.general.iocage
+   host: 10.1.0.73
+   user: admin
+   env:
+     CRYPTOGRAPHY_OPENSSL_NO_LEGACY: 1
+   sudo: true
+   sudo_preserve_env: true
+
+In this case, make sure the sudo tag *SETENV* is used:
```

Comment:
In Ansible documentation, we generally use code style and not italics for these things:
```suggestion
In this case, make sure the sudo tag ``SETENV`` is used:
```